packages/nodes-base/nodes/LocalFileTrigger/LocalFileTrigger.node.ts

```diff
@@ -173,23 +218,19 @@ export class LocalFileTrigger implements INodeType {
 		const triggerOn = this.getNodeParameter('triggerOn') as string;
```

Comment:
As triggerOn does not directly impact the watching of files or files, we should consider to switch from "files or folders" to "path" and maybe even to the possibility to watch multiple paths.

Reply:
Files and folders could be unified as path.
If the "Watch for" would be required and empty by default, an active choice by the user would be required.
Backwards compatibility could be done by setting "File change" if "Watch for" is empty, which then could only occur if you have it already configured.

![Screenshot from 2022-12-14 09-47-48](https://user-images.githubusercontent.com/13757952/207549188-1c5f8fd0-8003-491b-b6f1-9d1de1c4d3d6.png)

---

Another possibility would be to modify it like this, but this would be a breaking change, which I would try to avoid.

![Screenshot from 2022-12-14 09-36-04](https://user-images.githubusercontent.com/13757952/207547914-b025beda-f6fb-4a0f-aa14-9fc1ac568950.png)

Reply:
@Joffcom what's your opinion on this?

Reply:
Do you agree with the additional options provided?

What do you think about merging "files" and "folders" into "path"? I would like to do that, because this would simplify the UI and from a user perspective it is not really clear that the only difference is that "watching for files" means, "watching for change events" also on directories. For example if you add a directory as file to watch, this looks like in the following screenshot:
![Screenshot from 2023-05-09 16-55-58](https://github.com/n8n-io/n8n/assets/13757952/a2fc91f2-94d5-499b-a691-bdbf7f195faa)

Reply:
@Joffcom would be nice if you could comment these two points. That would help me to finalize the PR.